guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -103,7 +107,6 @@ endif::[]
 If you are installing {ProductName} as a virtual machine hosted on {oVirt}, you must also enable the *Red{nbsp}Hat Common* repository, and install {oVirt} guest agents and drivers.
```

Comment:
This note now also shows up for Debian, but the link doesn't contain any instructions for Debian. I'd hide it.

Reply:
Addressed :+1: